documentation/Plugins-System-Preview-Only/Plugins-Glossary-Preview-Only.md

```diff
@@ -0,0 +1,29 @@
+## Glossary & Core Concepts
+**Plugin** - Conceptually, a plugin is dynamically loaded software that can process traces. A plugin may contain one or more processing sources.  In our current system,  unlike processing source,  plugin doesn't have a tangible form.  As we introduce *plugin package* to our ecosystem, plugin becomes tangible. See **Plugin Package** for explanation.
+
+**Processing Source** - An entry point for a plugin. A processing source declares a human-readable name and description, defines the data sources it supports, creates the data processor that will process instances of supported data sources and advertises the tables that may be built in response to data sources being processed
+
+**Plugin Package** - A zip file with a custom extension (e.g. *.plugin*) containing the content of a plugin along with a metadata file that describes the plugin(s).  A plugin package may contain only one plugin. In other words, whatever is packaged into a plugin package by the the plugin author will be called **a** plugin in our new ecosystem. For v1, a plugin package will be self-contained in the way that it doesn't have dependencies on any other non-plugin packages. For example, a plugin must include Newtonsoft.Json if it requires it for JSON parsing.
```

Comment:
If a package may contain only one plugin, remove the (s) from the preceding sentence